src/app/PCM/templates/tpl_vertical_cards/cardItem.js

```diff
@@ -1,88 +1,108 @@
-import { Box, HStack, VStack } from '@chakra-ui/react'
-import { equals, map } from 'ramda'
+import { AspectRatio, Box, HStack, Text } from '@chakra-ui/react'
+import { filter, includes, map } from 'ramda'
 
+import AmenityField from 'app/PCM/templates/tpl-detail-view/tpl-property-detail-view/amenity-field.js'
 import Attribute from 'app/BE/attribute'
-import ContextMenu from 'app/BE/context'
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
-import { faEllipsisV } from '@fortawesome/free-solid-svg-icons'
+import { selectCode } from 'redux/db/selectors'
+import { useIsMobile } from 'utils/hooks'
+import { useSelector } from 'react-redux'
 
 const CardItem = ({ mappedValues, baseEntityCode, actions, sbeCode, primaryColor }) => {
+  const isMobile = useIsMobile()
+  const hasDisplayImage = includes('_LNK_PROPERTY__PRI_IMAGES')(mappedValues)
+  const attrName = useSelector(selectCode(baseEntityCode, 'PRI_CREATED'))?.attributeName || ''
+
+  const suburb =
+    useSelector(selectCode(baseEntityCode, '_LNK_PROPERTY__PRI_ADDRESS_SUBURB'))?.value || ''
+  const state =
+    useSelector(selectCode(baseEntityCode, '_LNK_PROPERTY__PRI_ADDRESS_STATE'))?.value || ''
+
+  const location = !!suburb && !!state ? `${suburb}, ${state}` : suburb || state
```

Comment:
what if suburb and state are both undefined? Probably a good idea to default to empty string

Reply:
fixed if both suburb and state doesn't have value